types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
Strange I got it from angular's definitions
![screen shot 2017-07-11 at 11 22 17 am](https://user-images.githubusercontent.com/2039539/28083742-4298aae6-662b-11e7-8b6c-bb563ffb835c.png)
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/angular/index.d.ts

Maybe thats not the correct way then I'm not too sure. I just want to make sure it works with imports.